src/main/java/seedu/address/model/pet/Food.java

```diff
@@ -29,11 +29,26 @@
     public Food(String foodName, int foodAmount) {
         requireNonNull(foodName);
         checkArgument(isValidFoodName(foodName), MESSAGE_NAME_CONSTRAINTS);
-        this.foodName = foodName;
+        this.foodName = formatFoodName(foodName);
         checkArgument(isValidFoodAmount(foodAmount), MESSAGE_AMOUNT_CONSTRAINTS);
         this.foodAmount = foodAmount;
     }
 
+    /**
+     * Transfer food name in to the format of "Xxx Xxx ...".
+     * @param foodName The food name passed in by user, to be
```

Comment:
to be?

Reply:
Here I mean to format the name/food name/species, such that each of the word start with an upper case while the rest are lower case. i.e. If user enter "john doe" or "JOHN DOE", pet name will always be "John Doe". same for species and foodname. 

This is to minimise possible confusion of species/name/...  If the user enter same thing with different letter case, and we do not standardize the format, then they will be treated as different objects. This might cause inaccuracy in stats and display i.

Reply:
actually what I meant was that the @param line does not seem complete. There is the "to be" hanging there. Same for some other comments I raised haha.